pyanaconda/install.py

```diff
@@ -253,24 +253,9 @@ def doInstall(storage, payload, ksdata, instClass):
         if iutil.getSysroot() != iutil.getTargetPhysicalRoot():
             blivet.setSysroot(iutil.getTargetPhysicalRoot(),
                               iutil.getSysroot())
-
```

Comment:
I guess this part is specific only to ostree install and I guess it is mainly because of ``if iutil.getSysroot() != iutil.getTargetPhysicalRoot():``.

If it is that case, could you please add the comment explaining why this is working to this commit.

I understand that this should be done by us some time ago when this was created, so or so it could be nice improvement of this commit.